CMakeLists.txt

```diff
@@ -65,6 +65,10 @@ set(ZSF_SOURCE_FILES
     src/load_time_averaged.c
     src/sealock.h
     src/sealock.c
+    src/ini.h
+    src/ini.c
+    src/zsf_config.h
+    src/zsf_config.c
```

Comment:
Should these get a separate folder? And could you rename ini to ini_reader or read_ini?

Reply:
Good suggestion, I guess the same would go for the csv(-related) files (Though a bit out of scope for this issue)